docs/requirements.txt

```
@@ -1,3 +1,3 @@
 pyenchant
-sphinx
+Sphinx==4.2.0
```

IMO, there is no need to pin a specific version. I would use `​>=3.1.0`
```suggestion
Sphinx>=3.1.0
```
`Sphinx` 3.1.0 is the first version which officially support Python 3.8 (supported by all supported versions of Django).

---

OK, done. (RTD's docs are quite keen on explicit pins, but that's not really how we do things, so … 🤷)